webpack.config.mjs

```diff
@@ -85,6 +85,7 @@ const config = {
 			),
 		},
 		extensions: ['.js', '.ts', '.tsx', '.jsx'],
+		conditionNames: ['development', 'import', 'default'],
```

Comment:
This tells webpack to use the development field defined in package.json exports which for @guardian/commercial points to the source typescript.